eng/native/configurecompiler.cmake

```diff
@@ -211,8 +211,8 @@ if (CLR_CMAKE_ENABLE_SANITIZERS)
       # Disable the use-after-return check for ASAN on Clang. This is because we have a lot of code that
```

Comment:
Should this comment still say CLang

Reply:
Yep. Throughout the CMake logic, we refer to both Clang variants as Clang. We don't refer to AppleClang specifically unless we need to do something specific for it (which we haven't so far).